### DISCUSSION TITLE
Question:  Call the add() event in custom comp?

### DISCUSSION THREAD

**Discussion by lajbel** [replit/kaboom#142] [Q&A] [answered]
I'm writing a component and everything seems to be that the component's add() function is only called when it is created, is there a way to get it to be called again? I have already tried using `obj.use(customComp())` and `readd(obj)`

**Comment by slmjkdbtl** [comment]
Yeah I'm aware of this 👍 will fix today

**Comment by slmjkdbtl** [comment, marked as answer]
Can you test with the latest beta release if it's fixed?

**Reply by lajbel** [reply]
Fixed!

**Comment by slmjkdbtl** [comment]
Just for the heads up, I don't think `readd()` should trigger `"add"` event (it's mainly used to get some existing obj to draw on top again) but it is currently, removing it.
